iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -866,6 +881,18 @@ struct MakeDispatchWorkgroupsOp : public RewritePattern {
       return failure();
     }
 
+    // If this is a standalone fill op, we don't need to create a dispatch
```

Comment:
If this was done after https://github.com/google/iree/pull/5576 , then you could check for "uses" of a `FillOp` and if there are no `linalgOp` uses do this conversion. That would be better suited.